From 6a03100c189d00e3a8235215392465b5b877ba8f Mon Sep 17 00:00:00 2001
From: Krzysztof Gajdemski <songo@debian.org.pl>
Date: Thu, 21 Mar 2013 11:40:06 +0100
Subject: [PATCH 8/9] ipvsadm: Fix wrong format of -o option in FMT_RULE
 listing

'ipvsadm -S' listed one-packet scheduling option in wrong format
('ops' instead of '--ops' or '-o') preventing any service with OPS
feature from restoring using 'ipvsadm -R'.  Now we use '-o' which
works well with save/restore commands.

Signed-off-by: Krzysztof Gajdemski <songo@debian.org.pl>
Signed-off-by: Simon Horman <horms@verge.net.au>
---
 ipvsadm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ipvsadm.c b/ipvsadm.c
index d13524f..454287c 100644
--- a/ipvsadm.c
+++ b/ipvsadm.c
@@ -1487,7 +1487,7 @@ print_service_entry(ipvs_service_entry_t *se, unsigned int format)
 		if (se->pe_name[0])
 			printf(" pe %s", se->pe_name);
 		if (se->flags & IP_VS_SVC_F_ONEPACKET)
-			printf(" ops");
+			printf(" -o");
 	} else if (format & FMT_STATS) {
 		printf("%-33s", svc_name);
 		print_largenum(se->stats.conns, format);
-- 
1.8.1.4

